src/models/UserModel.js

```diff
@@ -7,21 +7,22 @@ export const createUser = async (firstname, lastname, email, password) => {
 	const values = [firstname, lastname, email, password];
 
 	try {
-		const result = await dbConnection.query(query, values);
+		const result = await pool.query(query, values);
```

Comment:
Use the `dbConnection` instead of `pool`. The reason we are doing this is that instead of using pool from the pg library, it's possible that we decide to change to another library. If that happens, we would have to make the change using pool.query to `somethingelse.query()` or even `somethingelse.someFunctionelse()`  and we would do that in all our model files, but if we use `dbConnection` which is an abstraction over the pool client, we would only make the change in one file - the file where `dbConnection` is defined

Also we have some other logic in the `dbConnection` object, which you would be getting rid of by replacing it with `pool`.